docs/docs/guides/installation/knative.mdx

```diff
@@ -43,10 +43,14 @@ create a secret with your certificates for TLS termination
 kubectl apply secret -f certs.yaml
 ```
 
+
 ## Start ZITADEL with Knative 
 ```bash
+
+# start zitadel
+
 kn service create zitadel \
---image ghcr.io/zitadel/zitadel:v2.0.0-v2-alpha.30-amd64 \
+--image ghcr.io/zitadel/zitadel:latest \
```

Comment:
I am not a fan of using latest. I would prefer something like v2-alpha or v2

We could extend the tagging here https://github.com/zitadel/zitadel/blob/v2-alpha/.goreleaser.yaml#L35

Reply:
for the moment i changed to the latest specific version (alpha33)

but i agree we could / should change to something like v2